requirements/runtime.txt

```diff
@@ -0,0 +1,10 @@
+imagecorruptions
```

Comment:
Nice work! `imagecorruptions` may also be made optional.

Reply:
done